src/content/1.7/basics/keeping_up-to-date/upgrade.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Upgrade
```

Comment:
I think we can change this to:

title: "How to upgrade PrestaShop 1.6 to PrestasShop 1.7
menuTitle: "Upgrade"

Reply:
No, this process should be used for minor upgrades first. Upgrading from 1.6 to 1.7 is possible, but not recommended in these tutorials.